sensors/triggers/kafka/kafka.go

```diff
@@ -216,6 +216,7 @@ func (t *KafkaTrigger) Execute(ctx context.Context, events map[string]*v1alpha1.
 
 	msg := &sarama.ProducerMessage{
 		Topic:     trigger.Topic,
+		Key:       sarama.StringEncoder(t.Trigger.Template.Name),
```

Comment:
No, I can remove this.

One benefit of setting the trigger name as the key is that downstream messages will arrive in order, whereas if no key is specified two trigger invocations may end up on different partitions and therefore may be processed out of order. However, since the key was not set in the first place I'm guessing this hasn't been much of an issue in practice.